.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing Guide
+
+This guide is for anyone who wishes to contribute code to Bhagavad Gita API.
+
+Thank you for your interest and welcome here!
+
+To work on this project you will need the following software installed in your machine.
+
+- git (version control)
+- python3.8 or above
+- poetry (package management)
+- make (command line utils)
+- docker (optional, if you want to build docker images)
+
+1. First of all fork and clone this repo. Checkout a new branch to start working.
+For more information read
+[GitHub's Docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+for beginners.
+
+2. If you don't already have `poetry`, then [install it](https://python-poetry.org/docs/#installation).
+Move into the project directory and run the following commands.
+
+    ```shell
+    poetry config virtualenvs.in-project true
+    poetry install
+    ```
+
+3. The virtual environment will be created in a `.venv` folder inside your
+project directory.
+In your code editor set the python interpretor path to `./.venv/bin/python`
+
+4. Activate poetry shell.
+
+    ```shell
+    poetry shell
+    ```
+
+5. Install pre-commit hooks.
+
+    ```shell
+    pre-commit install
+    ```
+
```

Comment:
Can we add a step for setting up the environment variables before running the app? Otherwise, the user will get an error.